docs/reference/tab-widgets/troubleshooting/snapshot/corrupt-repository.asciidoc

```diff
@@ -0,0 +1,219 @@
+// tag::cloud[]
+Fixing the corrupted repository will entail making changes in multiple deployments
+that write to the same snapshot repository.
+Only one deployment must be writing to a repository. The deployment 
+that will keep writing to the repository will be called the "primary" deployment (the current cluster), 
+and the other one(s) where we'll mark the repository as read-only the "secondary" 
+deployments.
+
+First mark the repository as read-only on the secondary deployments:
+
+**Use {kib}**
+
+//tag::kibana-api-ex[]
+. Log in to the {ess-console}[{ecloud} console].
++
+
+. On the **Elasticsearch Service** panel, click the name of your deployment. 
++
+
+NOTE:
+If the name of your deployment is disabled your {kib} instances might be
+unhealthy, in which case please contact https://support.elastic.co[Elastic Support].
+If your deployment doesn't include {kib}, all you need to do is 
+{cloud}/ec-access-kibana.html[enable it first].
+
+. Open your deployment's side navigation menu (placed under the Elastic logo in the upper left corner)
+and go to **Stack Management > Snapshot and Restore > Repositories**.
++
+[role="screenshot"]
+image::images/repositories.png[{kib} Console,align="center"]
+
+. The repositories table should now be visible. Click on the pencil icon at the 
+right side of the repository to be marked as read-only. On the Edit page that opened 
+scroll down and check "Read-only repository". Click "Save".
+Alternatively if deleting the repository altogether is prefered select the checkbox
+at the left of the repository name in the repositories table and click the 
+"Remove repository" red button at the top left of the table.
+
+At this point, it's only the primary (current) deployment that has the repository marked
+as writeable.
+{es} sees it as corrupt, so the repository needs to be removed and added back so that 
+{es} can resume using it:
+
+Note that we're now configuring the primary (current) deployment.
+
+. Open the primary deployment's side navigation menu (placed under the Elastic logo in the upper left corner)
+and go to **Stack Management > Snapshot and Restore > Repositories**.
++
+[role="screenshot"]
+image::images/repositories.png[{kib} Console,align="center"]
+
+. Get the details for the repository we'll recreate later by clicking on the repository
+name in the repositories table and making note of all the repository configurations
+that are displayed on the repository details page (we'll use them when we recreate
+the repository). Close the details page using the "X Close" link at
+the bottom left of the page.
```

Comment:
I think taking out the text will help in the future if it ever changes (totally optional though)

```suggestion
the repository). Close the details page using the link at
the bottom left of the page.
```